tests/test_job_management.py

```diff
@@ -1162,6 +1168,127 @@ def test_email_very_long_request_url(dispatcher_long_living_fixture, dispatcher_
 This might be fixed in a future release.""" in email_data
 
 
+@pytest.mark.not_safe_parallel
+@pytest.mark.parametrize("use_scws_value", ['form_list', 'user_file', 'no', None])
+@pytest.mark.parametrize("passing_scw_list", [True, False])
+def test_email_scws_list(dispatcher_live_fixture,
+                         dispatcher_local_mail_server,
+                         use_scws_value,
+                         passing_scw_list):
+    DispatcherJobState.remove_scratch_folders()
+
+    server = dispatcher_live_fixture
+    logger.info("constructed server: %s", server)
+
+    # let's generate a valid token
+    token_payload = {
+        **default_token_payload,
+        "roles": "unige-hpc-full, general",
+    }
+    encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+
+    params = {
+        'query_status': "new",
+        'product_type': "dummy",
+        'query_type': "Real",
+        'instrument': 'empty-async',
+        'use_scws': use_scws_value,
+        'token': encoded_token
+    }
+
+    if use_scws_value == 'user_file':
+        scw_list_file_obj = None
+        if passing_scw_list:
+            file_path = DispatcherJobState.create_scw_list_file(list_length=5)
+            scw_list_file = open(file_path).read()
+            scw_list_file_obj = {"user_scw_list_file": scw_list_file}
+        # we are supposed to send a file, so it has to be a post request
+        jdata = ask(server,
+                    params,
+                    method='post',
+                    max_time_s=150,
+                    expected_query_status=None,
+                    expected_status_code=None,
+                    files=scw_list_file_obj
+                    )
+    elif use_scws_value == 'no':
+        # no list should be passed
+        jdata = ask(server,
+                    params,
+                    max_time_s=150,
+                    expected_query_status=None,
+                    expected_status_code=None
+                    )
+    else:
+        if passing_scw_list:
+            params['scw_list'] = [f"0665{i:04d}0010.001" for i in range(5)]
```

Comment:
oh, by the way, we experienced another variation: sometimes it's a list, and sometimes it's coma-separated string. Good to test this here!

Reply:
Ok! I will include

Reply:
Is that a case for the api?